pmm-app/src/pmm-qan/panel/QueryAnalytics.tsx

```diff
@@ -57,8 +57,12 @@ const QueryAnalyticsPanel: FC = () => {
   );
 };
 
-export default (props) => (
-  <UrlParametersProvider {...props}>
-    <QueryAnalyticsPanel />
-  </UrlParametersProvider>
-);
+export default (props) => {
+  console.log('grafana props', props);
```

Comment:
Seems like this was added again for some reason but I know you deleted this console.log and mande the return implicit